matching/.env

```
@@ -0,0 +1,9 @@
+PINECONE_API_KEY=pcsk_3H3jXT_JUDTjLHX49Hcm3NvrFpv4hTk7xsFPF7Y5yDFHY7rEjJ5dygVJZo5GtS4ATedSbz
+PINECONE_INDEX_HOST=https://tea-mate-matching-l0a94ax.svc.aped-4627-b74a.pinecone.io
+PINECONE_ENVIRONMENT=us-east-1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PINECONE_ENVIRONMENT key should go before the PINECONE_INDEX_HOST key

(UnorderedKey)